submitted_models/coro_rocky_sensor_config_3/model.sdf

```diff
@@ -0,0 +1,945 @@
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+      <visual name="camera_lens_front_visual">
+        <pose>0.105 0.03 0.235 0 1.5707963267948966 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.011</length>
+            <radius>0.005</radius>
+          </cylinder>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+
+      <visual name="camera_rear_visual">
+        <pose>0.085 -0.03 0.235 0 0 0</pose>
+        <geometry>
+          <box>
+            <size>0.01 0.03 0.03</size>
+          </box>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+      <visual name="camera_lens_rear_visual">
+        <pose>0.075 -0.03 0.235 0 1.5707963267948966 0</pose>
+        <geometry>
+          <cylinder>
+            <length>0.011</length>
+            <radius>0.005</radius>
+          </cylinder>
+        </geometry>
+        <material>
+          <ambient>0.0 0.0 0.0 1.0</ambient>
+          <diffuse>0.0 0.0 0.0 1.0</diffuse>
+        </material>
+      </visual>
+
+
+      
+      <!-- Real robot has 2 x B0203 with LN008 each with a 180 deg hor x 130 deg vert FOV
+           Since fisheye cameras are not modeled, wide HD cameras are used in the model -->
+      <sensor name="camera_front" type="camera">
+        <pose>0.111 0.03 0.235 0 0 0</pose>
+        <always_on>1</always_on>
+        <update_rate>30</update_rate>
+        <camera name="camera_front">
+          <horizontal_fov>2.4</horizontal_fov>
+          <lens>
+            <intrinsics>
+              <!-- fx = fy = width / ( 2 * tan (hfov / 2 ) ) -->
+              <fx>248.819</fx>
+              <fy>248.819</fy>
+              <!-- cx = ( width - 1 ) / 2 -->
+              <cx>639.5</cx>
+              <!-- cy = ( height - 1 ) / 2 -->
+              <cy>359.5</cy>
+              <s>0</s>
+            </intrinsics>
+          </lens>
+          <distortion>
+            <k1>0.0</k1>
+            <k2>0.0</k2>
+            <k3>0.0</k3>
+            <p1>0.0</p1>
+            <p2>0.0</p2>
+            <center>0.5 0.5</center>
+          </distortion>
+          <image>
+            <width>1280</width>
+            <height>720</height>
+            <format>R8G8B8</format>
+          </image>
+          <clip>
+            <near>0.01</near>
+            <far>300</far>
+          </clip>
+          <noise>
+            <type>gaussian</type>
+            <mean>0</mean>
+            <stddev>0.007</stddev>
+          </noise>
+        </camera>
+      </sensor>
+
+      <sensor name="camera_rear" type="camera">
+        <pose>0.069 -0.03 0.235 0 0 3.141592653589793</pose>
+        <always_on>1</always_on>
+        <update_rate>30</update_rate>
+        <camera name="camera_front">

```

Comment:
<camera name="camera_rear">

Reply:
Thank you.  I have fixed the camera name.